sklearn/datasets/samples_generator.py

```diff
@@ -1172,9 +1172,19 @@ def make_sparse_spd_matrix(dim=1, alpha=0.95, norm_diag=False,
         If None, the random number generator is the RandomState instance used
         by `np.random`.
 
+    largest_coef : float between 0 and 1, optional (default=0.9)
+        The value of the largest coefficient.
+
+    smallest_coef : float between 0 and 1, optional (default=0.1)
+        The value of the smallest coefficient.
+
+    norm_diag : boolean, optional (default=False)
+        Whether to normalize the output matrix via diagonal normalization
```

Comment:
Yeah... The output after this operation, makes the leading diagonal to all ones...